Models/Certificate.swift

```diff
@@ -226,15 +246,25 @@ private struct CertificateV1_1 : Certificate {
         // Get any key properties on the Certificate object
         guard let certificateData = json["certificate"] as? [String: AnyObject],
             let title = certificateData["title"] as? String,
-            let subtitleMap = certificateData["subtitle"] as? [String : String],
+            let subtitleMap = certificateData["subtitle"] as? [String : AnyObject],
             let certificateImageURI = certificateData["image"] as? String,
             let certificateIdString = certificateData["id"] as? String,
             let certificateIdUrl = URL(string: certificateIdString),
             let description = certificateData["description"] as? String else {
             return nil
         }
         let certificateImage = imageData(from: certificateImageURI)
-        let subtitle = subtitleMap["display"] == "FALSE" ? nil : subtitleMap["content"]
+        var subtitle : String? = nil
+        if let subtitleDisplay = subtitleMap["display"] as? Bool {
+            if subtitleDisplay {
+                subtitle = (subtitleMap["content"] as? String)!
+            }
+        } else {
+            let subtitleDisplayStr = subtitleMap["display"] as! String
+            if subtitleDisplayStr.uppercased() == "TRUE" {
+                subtitle = (subtitleMap["content"] as? String)!
+            }
+        }
```

Comment:
You can combine some of these clauses. You also don't need the force-unwrap (`!`) since `subtitle` is already an optional string.

```
if let subtitleDisplay = subtitleMap["display"] as? Bool, 
    subtitleDisplay {
   subtitle = subtitleMap["content"]
} else if let subtitleDisplayStr = subtitleMap["display"] as? string, 
    subtitleDisplayStr.uppercased() == "TRUE" {
  subtitle = subtitleMap["content"] as? String
}
```

Also note, I changed one cast to `as?` instead of `as!` in the code above.